src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx

```diff
@@ -109,20 +109,11 @@ export const MeetTheSpecialists: React.FC = () => {
                 SPECIALISTS.filter(i => i.specialty === pill.type)
               )
             }}
-            style={
-              selectedSpecialty === pill.type
-                ? {
-                    color: "white",
-                    backgroundColor: "black",
-                    borderColor: "black",
-                  }
-                : undefined
-            }
```

Comment:
I think you can remove the `hover={false}` as well and add a selected state using `selected={selectedSpecialty === pill.type}`